.github/workflows/build-and-test.yml

```diff
@@ -56,6 +56,14 @@ jobs:
                   git fetch --prune --unshallow
                   git submodule -q update --init --recursive
 
+            - name: Setup nuget cache
+              uses: actions/cache@v2
+              id: nuget-cache
+              with:
+                  path: ~/.nuget
+                  key: ${{ runner.os }}-nuget-${{ hashFiles('**/ImageSharp*.csproj') }}
```

Comment:
you will need to add the `*.props`/`*.targets` too as we define most of our package versions in those shared locations i.e. https://github.com/SixLabors/ImageSharp/blob/0e0dc2ae9cafcdf5bde9d185919cf073ddf4f186/Directory.Build.targets#L19-L37

Reply:
Thanks. Added with next commit.